src/Workspaces/Remote/ServiceHub.CoreComponents/CoreComponents.Shared.targets

```diff
@@ -158,8 +158,8 @@
 
       <!-- the only assembly we need under runtime folder (runtimes\win-x64\native\e_sqlite3.dll) is handled by the vsix project directly -->
       <_ExcludedFiles Include="$(PublishDir)runtimes\**\*.*" />
-
-      <_ExcludedFiles Condition="'$(_PublishRuntimeLibraries)' == 'false'" Include="@(_RuntimeLibrariesInPublishDir)" />
+      <!-- Always include BCL in the vsix for CI build to avoid breakage. Since VS used on CI machines usually lags behind our insertion target -->
+      <_ExcludedFiles Condition="'$(_PublishRuntimeLibraries)' == 'false' and '$(ContinuousIntegrationBuild)' != 'true'" Include="@(_RuntimeLibrariesInPublishDir)" />
```

Comment:
I have merged the change after this, which is similar to how we use `Roslyn.VisualStudio.Setup.Dependencies`